.github/CODEOWNERS

```diff
@@ -44,9 +44,9 @@
 #  Folders where anyone with write access to the repo can approve Pull Requests
 # ================================================
 
-/docs/          @SAP/spartacus-project-team
+/docs/
 
-/projects/storefrontapp-e2e-cypress/          @SAP/spartacus-project-team
+/projects/storefrontapp-e2e-cypress/
```

Comment:
The folders now have no specific owners, so no emails will be sent.
Having no owners was the original intent anyways.